api/plugins/module_utils/api_client.py

```diff
@@ -387,6 +387,51 @@ def remove_project_notification(self, project, notification, type):
 
         return self.make_api_call(json.dumps(query) % (project, notification, type))
 
+    def user_add_group(email, group_name, role):
+        query = {
+            'query': """mutation group(
+                $email: String!
+                $group: String!
+                $role: String!
```

Comment:
You've got `$role` defined, but do not provide it in variables and `$group` is defined as argument, but `$name` provided in variables.